src/main.js

```diff
@@ -699,14 +700,64 @@ function getNodePosition(node){
   return d3.transform(d3.select(node).attr("transform")).translate;
 }
 
+
 function addEleContent(e) {
-  var newNodeAddress = "http://webstrates.ucsd.edu/" + hoveredEle;
+  let newNodeAddress = "http://webstrates.ucsd.edu/" + hoveredEle;
+  let wrapper = document.createElement('div');
+  wrapper.setAttribute("id", "addContentWrapper");
+  
+  let toFrame = '<iframe id="addWindow" src="'+ newNodeAddress + '" style="width: 95%; height: 350px;"><p>ERROR: Your browser does not support iframes.</p></iframe>';
+  let addNoteButton = '<button type="button" id="addNoteBtn" onclick="appendNote()">Add Sticky Note</button> ';
+  let addPicButton = '<input type="file" id="addPicBtn" onchange="appendPic()"> ';
+  let closeButton = '<button type="button" id="closeWindowBtn" onclick="closeContentWindow()">Close Content Window</button> ';
+  wrapper.innerHTML = toFrame + addNoteButton + addPicButton + closeButton;
+  document.getElementById("content_container").appendChild(wrapper);
+
+  addEleToList(e);
+    
+  addWindowOpen = true;
+
+}
+
+function closeContentWindow() {
+  let wrapper = document.getElementById("addContentWrapper");
+  let childOne = document.getElementById("addWindow");
+  let childTwo = document.getElementById("addNoteBtn");
+  let childFour = document.getElementById("addPicBtn");
+  let childThree = document.getElementById("closeWindowBtn");
+  childOne.parentNode.removeChild(childOne);
+  childTwo.parentNode.removeChild(childTwo);
+  childFour.parentNode.removeChild(childFour);
+  childThree.parentNode.removeChild(childThree);
+  wrapper.parentNode.removeChild(wrapper);
+  addWindowOpen = false;
+}
 
-  var appendElement = '<div class="note" contenteditable="true" style="position: absolute;left: 8px;top: 8px;width: 200px;min-height: 200px;padding: 16px;box-shadow: 5px 5px 10px gray;background-color: rgb(255, 255, 150);font-size: 24pt;word-wrap: break-word;"></div>'
+function appendNote() {
+  let appendElement = '<div class="note" contenteditable="true" style="left: 8px;top: 8px;width: 235px;min-height: 100px;padding: 16px;box-shadow: 5px 5px 10px gray;background-color: rgb(255, 255, 150);font-size: 12pt;word-wrap: break-word;"></div><br>';    
```

Comment:
Define the static file inside the css file. 
I think "textarea" is better than contenteditable.

Reply:
Done